src/pages/Index.vue

```diff
@@ -2,18 +2,115 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    <g-image alt="Example image" src="~/favicon.png" width="135" />
+    <header class="header">
+        <b-container class=".b-container">
+        <b-row>
+          <b-col lg="6" md="6" sm="12">
```

Comment:
As the 2 columns would have same size, I would remove the `lg`, `md` and `sm` here

Reply:
Okay, I'll do that. Thank you!